src/Http/Http.Abstractions/src/ConnectionInfo.cs

```diff
@@ -8,23 +8,45 @@
 
 namespace Microsoft.AspNetCore.Http
 {
+    /// <summary>
```

Comment:
Shoot: https://github.com/dotnet/aspnetcore/pull/26358. I had some of these documented in an earlier pass. I should have clarified it in the docs